lib/core-integration/src/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -1474,6 +1491,91 @@ spec = do
               length <$> [txs1, txs2] `shouldSatisfy` all (== 0)
 
+    it "TRANS_DELETE_01 - Single Output Transaction" $ \ctx -> do
+        (wa, wb) <- (,) <$> fixtureWallet ctx <*> fixtureWallet ctx
+        addrs <- listAddresses ctx wb
+
+        let amt = 1
+        let destination = (addrs !! 1) ^. #id
+        let payload = Json [json|{
+                "payments": [{
+                    "address": #{destination},
+                    "amount": {
+                        "quantity": #{amt},
+                        "unit": "lovelace"
+                    }
+                }],
+                "passphrase": "cardano-wallet"
+            }|]
+        let (feeMin, feeMax) = ctx ^. feeEstimator $ TxDescription
+                { nInputs = 1
+                , nOutputs = 1
+                }
+
+        r <- request @(ApiTransaction t) ctx (postTxEp wa) Default payload
+        verify r
+            [ expectSuccess
+            , expectResponseCode HTTP.status202
+            , expectFieldBetween DSL.amount (feeMin + amt, feeMax + amt)
+            , expectFieldEqual DSL.direction Outgoing
+            , expectFieldEqual DSL.status Pending
+            ]
+
+        let (ApiTransaction txId _ _ _ _ _ _ _ _) = getFromResponse Prelude.id r
+
+        ra <- request @ApiWallet ctx (getWalletEp wa) Default Empty
+        verify ra
+            [ expectSuccess
+            , expectFieldBetween balanceTotal
+                ( faucetAmt - feeMax - amt
+                , faucetAmt - feeMin - amt
+                )
+            , expectFieldEqual balanceAvailable (faucetAmt - faucetUtxoAmt)
+            ]
+
+        rDel <- request @ApiTxId ctx (deleteTxEp wa (ApiTxId txId)) Default Empty
+        expectResponseCode @IO HTTP.status204 rDel
+
+        rb <- request @ApiWallet ctx (getWalletEp wa) Default Empty
+        verify rb
+            [ expectSuccess
+            , expectFieldEqual balanceTotal faucetAmt
+            , expectFieldEqual balanceAvailable faucetAmt
+            ]
+
+        rc <- request @ApiWallet ctx (getWalletEp wb) Default Empty
+        verify rc
+            [ expectSuccess
+            , expectEventually ctx getWalletEp balanceAvailable (faucetAmt + amt)
+            ]
+
+        verify rb
+            [ expectEventually ctx getWalletEp balanceAvailable (faucetAmt - feeMax - amt)
+            ]
+
+        rDel1 <- request @ApiTxId ctx (deleteTxEp wa (ApiTxId txId)) Default Empty
+        expectResponseCode @IO HTTP.status404 rDel1
+        expectErrorMessage (errMsg404NoPendingAnymore (toUrlPiece (ApiTxId txId))) rDel1
+
+        let (ApiT (Hash load)) = txId
+        let txIdCorrupted = ApiT $ Hash $ BS.snoc (BS.tail load) (BS.head load)
+
+        rDel2 <-
+             request @ApiTxId ctx (deleteTxEp wa (ApiTxId txIdCorrupted)) Default Empty
+        expectResponseCode @IO HTTP.status404 rDel2
+        expectErrorMessage (errMsg404CannotFindTx (toUrlPiece (ApiTxId txIdCorrupted))) rDel2
```

Comment:
This small scenario should also be in a separate test case.

Reply:
done